File: URI Programs/C/1045.c
#include <stdio.h>

int main(){
	double a, b, c, temp;
	scanf("%lf %lf %lf", &a, &b, &c);
	if (a < b){
    	temp = a;
    	a = b;
    	b = temp;
    }
    if (b < c){
		temp = b;
		b = c;
		c = temp;
	}
    if (a < b){
		temp = a;
		a = b;
		b = temp;
	}
	if (a >= (b + c)){
		printf("NAO FORMA TRIANGULO\n");
	} else{
		if (pow(a, 2) == pow(b, 2) + pow(c, 2)){
			printf("TRIANGULO RETANGULO\n");
		} else if (pow(a, 2) > pow(b, 2) + pow(c, 2)){
			printf("TRIANGULO OBTUSANGULO\n");
		} else {
			printf("TRIANGULO ACUTANGULO\n");
		}
		if (c == b && b == a){
			printf("TRIANGULO EQUILATERO\n");
		} else if (c == b || b == a){
			printf("TRIANGULO ISOSCELES\n");
		}
	}
	return 0;
}

File: URI Programs/Python/1045.py
x, y, z = map(float, input().split())

L = []
L.append(x)
L.append(y)
L.append(z)
L = sorted(L)

A = L[2]
B = L[1]
C = L[0]

if A >= B + C :
  print("NAO FORMA TRIANGULO")
else:
  if A**2 == (B**2 + C**2):
    print("TRIANGULO RETANGULO")
  elif A**2 > B**2 + C**2:
      print("TRIANGULO OBTUSANGULO")
      if A == B == C:
        print("TRIANGULO EQUILATERO")
      elif A == B or B == C or C == A:
        print("TRIANGULO ISOSCELES")
  elif A**2 < (B**2 + C**2):
    print("TRIANGULO ACUTANGULO")
    if A == B == C:
      print("TRIANGULO EQUILATERO")
    elif A == B or B == C or C == A:
      print("TRIANGULO ISOSCELES")